app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = @search.keyword(params[:name])
```

Comment:
Instead of relying on the callback_action, consider just calling the facade here. Also make sure to name things properly and not based on your old project! Can be named `FoodFacade` instead of TmdbSearch